content/docs/ref/comment.md

```diff
@@ -42,12 +42,12 @@ Any [generic option](/doc/ref) in addition to:
 - `--watch`: Watch for changes and automatically update the comment (doesn't
   exit, consider
   [appending `&` to run in the background](<https://en.wikipedia.org/wiki/Job_control_(Unix)#Implementation>)).
-- `--publish`: Upload any local images found in the Markdown report.
+- `--publish=false`: Don't upload any local images found in the Markdown report.
```

Comment:
Make it more explicit because it's a default-true flag

```suggestion
- `--publish=<true|false>`: Upload any local images found in the Markdown report.
```